mmv1/third_party/terraform/resources/resource_tags_location_tag_bindings.go.erb

```diff
@@ -0,0 +1,349 @@
+			url, err = addQueryParams(url, map[string]string{"pageToken": fmt.Sprintf("%s", res["nextPageToken"])})
+			if err != nil {
+				return handleNotFoundError(err, d, fmt.Sprintf("TagsLocationTagBinding %q", d.Id()))
+			}
+			resp, err := sendRequest(config, "GET", billingProject, url, userAgent, nil)
+			if err != nil {
+				return handleNotFoundError(err, d, fmt.Sprintf("TagsLocationTagBinding %q", d.Id()))
+			}
+			if resp == nil {
+				// Object isn't there any more - remove it from the state.
+				d.SetId("")
+				return nil
+			}
+			v, ok := resp["tagBindings"]
+			if !ok || v == nil {
+				return nil
+			}
+			pView = append(pView, v.([]interface{})...)
+			if token, ok := res["nextPageToken"]; ok {
+				pageToken = token.(string)
+			} else {
+				pageToken = ""
+			}
+		}
+	}
+
+	newMap := make(map[string]interface{}, 1)
+	newMap["tagBindings"] = pView
+
+	res, err = flattenNestedTagsLocationTagBinding(d, meta, newMap)
+	if err != nil {
+		return err
+	}
+
+	if err := d.Set("name", flattenNestedTagsLocationTagBindingName(res["name"], d, config)); err != nil {
+		return fmt.Errorf("Error reading LocationTagBinding: %s", err)
+	}
+	if err := d.Set("parent", flattenNestedTagsLocationTagBindingParent(res["parent"], d, config)); err != nil {
+		return fmt.Errorf("Error reading LocationTagBinding: %s", err)
+	}
+	if err := d.Set("tag_value", flattenNestedTagsLocationTagBindingTagValue(res["tagValue"], d, config)); err != nil {
+		return fmt.Errorf("Error reading LocationTagBinding: %s", err)
+	}
+
+	return nil
+}
+
+func resourceTagsLocationTagBindingDelete(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	userAgent, err := generateUserAgentString(d, config.userAgent)
+	if err != nil {
+		return err
+	}
+
+	billingProject := ""
+
+	url, err := replaceVars(d, config, "{{TagsLocationBasePath}}{{name}}")
+	if err != nil {
+		return err
+	}
+
+	var obj map[string]interface{}
+	log.Printf("[DEBUG] Deleting LocationTagBinding %q", d.Id())
+
+	// err == nil indicates that the billing_project value was found
+	if bp, err := getBillingProject(d, config); err == nil {
+		billingProject = bp
+	}
+
+	res, err := sendRequestWithTimeout(config, "DELETE", billingProject, url, userAgent, obj, d.Timeout(schema.TimeoutDelete))
+	if err != nil {
+		return handleNotFoundError(err, d, "LocationTagBinding")
+	}
+
+	err = tagsOperationWaitTime(
+		config, res, "Deleting LocationTagBinding", userAgent,
+		d.Timeout(schema.TimeoutDelete))
+
+	if err != nil {
+		return err
+	}
+
+	log.Printf("[DEBUG] Finished deleting LocationTagBinding %q: %#v", d.Id(), res)
+	return nil
+}
+
+func resourceTagsLocationTagBindingImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	config := meta.(*Config)
+	if err := parseImportId([]string{"(?P<location>[^/]+)/tagBindings/(?P<name>[^/]+)/tagValues/(?P<tag_value>[^/]+)"}, d, config); err != nil {
+		return nil, err
+	}
+
+	parent := d.Get("name").(string)
```

Comment:
Parent == name once the URL-encoded values are replaced with slashes? Why wouldn't this be name?

Reply:
Why do we have to handle escaped values anyways? Seems like we should remove them before storing the ID so users have actual full values rather than partially escaped strings in their state